modules/zivid/matrix4x4.py

```diff
@@ -0,0 +1,93 @@
+"""Contains Matrix4x4 class."""
+import pathlib
+import _zivid
+
+
+class Matrix4x4(_zivid.Matrix4x4):
+    """Matrix of size 4x4 containing 32-bit floats."""
+
+    def inverse(self):
+        """
+        Return the inverse of this matrix.
+
+        An exception is thrown if the matrix is not invertible.
+
+        Returns:
+            A new matrix, holding the inverse
+        """
+        return Matrix4x4(super().inverse())
+
+    def load(self, file_path):
+        """
+        Load the matrix from the given file.
+
+        Args:
+            file_path: path for the file to load the matrix from.
+        """
+        if isinstance(file_path, pathlib.Path):
+            file_path = str(file_path)
+        super().load(file_path)
+
+    def save(self, file_path):
+        """
+        Save the matrix to the given file.
+
+        Args:
+            file_path: path for the new file to save the matrix in.
+        """
+        if isinstance(file_path, pathlib.Path):
+            file_path = str(file_path)
+        super().save(file_path)
```

Comment:
You can just do `super().save(str(file_path))` because `str()` on a string does nothing :)

Reply:
Same in the load() method.

Reply:
It is possibly something else though. For example, in your sample `matrix.save(123)` would compile because `123` can be convertible to a string. But it should definitely not compile.

Reply:
Good point.
But it seems we already are pretty liberal on the string conversions here,
https://github.com/zivid/zivid-python/blob/17ad11301021cb4aac94f09892f86904bd42009d/modules/zivid/application.py#L31

so parhaps do the same here for consistency?

Reply:
Oh, then okay. Consistency is important.